### ISSUE TITLE
Frostmaw continously attacks nothing after other mob dies

### ISSUE THREAD

**Issue by DWX12** [BobMowzie/MowziesMobs#26]
After a Frostmaw has killed another mob, I've found that it will sometimes it will just keep attacking nothing.

On a side note, Frostmaws seem like they can be pushed around more easily than say the Wrought Knight. Noticed it when testing out mobs fighting each other with Ice and Fire. Cyclops' tend to just move the Frostmaw around, usually while it's still sleeping.

**Comment by BobMowzie** [comment]
I am not observing this issue. What other mods do you have installed? How did you get the frostmaw to target another mob?

**Comment by DWX12** [comment]
The Frostmaw usually attacks other mobs through the usual method of getting attacked itself as will other mobs such as when skeletons shoot spiders or zombies, etc. I have Ice and Fire (which I was primarily using when testing out your mobs). I was mainly using cyclops', but tried out dragons and a few other mobs.The cyclops usually goes after pretty much any non-passive mob and does either a kick attack or picks up and bites the other mob for massive damage (although I have it toned down some in my config).

It has been awhile since I did any testing and Ice and Fire has since updated twice in that period of time. However, I don't believe it has anything to do with the behavior of your mobs (programming is a strange world and I don't rule out any possibility), it's just the one I was testing it out with. I experienced this odd behavior with other mobs, with some being vanilla, if I recall correctly.

The next time I'm able to sit down and do some testing I will inform you on any more strange behaviors of your mobs.

Thank you again for all of your hard work and I will get back to you.
